code/modules/food_and_drinks/recipes/tablecraft/recipes_salad.dm

```diff
@@ -124,3 +124,17 @@
 	)
 	result = /obj/item/reagent_containers/food/snacks/salad/ricepudding
 	subcategory = CAT_SALAD
+
+/datum/crafting_recipe/food/bellpeppers_and_beef
+	name = "Bell Peppers and Beef"
+	reqs = list(
+		/obj/item/reagent_containers/glass/bowl = 1,
+		/obj/item/reagent_containers/food/snacks/grown/chili = 4,
+		/obj/item/reagent_containers/food/snacks/grown/onion = 1,
+		/obj/item/reagent_containers/food/snacks/grown/mushroom/plumphelmet = 1, // we dont have shiitakes so this will have to do
```

Comment:
Why not make this any mushroom type? Let a mean chef make this with fly amanita or liberty caps.